### ISSUE TITLE
Menu not accessible

### ISSUE THREAD

**Issue by Heydon** [TryGhost/Casper#288]
Some issues:

- No focus style
- A link is used as the button, producing the wrong role for screen reader users (and since the link doesn't point anywhere, it doesn't behave like a link either)
- The state (open / closed) is not communicated
- Focus is not moved to the menu when it is opened, meaning it is not reachable by keyboard.

Suggested HTML (elided):

```html
<button aria-expanded="false">
  <span class="word">Menu</span>
</button>
```
_(Note that `aria-expanded` should be switched to `true` on opening the menu)_.

So that the menu is reachable by keyboard, `focus()` should be applied to the first of the menu items on opening the menu. When pressing the close button, the Menu button should be refocused using `focus()`.

**Comment by MichielBijl** [comment]
The [WAI-ARIA Authoring Practices](http://w3c.github.io/aria-practices/) has some more information on how to make [menus](http://w3c.github.io/aria-practices/#menu) or [menu buttons](http://w3c.github.io/aria-practices/#menubutton) accessible.

**Comment by margemcm** [comment]
Hi there!  I'm working on this issue with some classmates.
We've gotten varying responses as to whether aria-expanded should be applied to the menu itself or the button, I was wondering if anyone had any input.

**Comment by Heydon** [comment]
@margemcm Hi! `aria-expanded` goes — perhaps unintuitively to begin with — on the button. That's so, when you focus the button, it tells you whether _the thing it controls_ is expanded or not.

Hope that helps!

**Comment by JohnONolan** [comment]
Master branch has now moved on to 2.0 design which no longer has any sort of animated/hidden mobile menu.
